src/components/Dashboard/RewardsModule.js

```diff
@@ -78,7 +85,10 @@ const RewardsModule = React.memo(function RewardsModule({
   const totalDisputesFees = totalArbitrableFees.add(totalAppealFees)
 
   // All dispute fees are sent to the treasury after being settled
-  const totalTreasuryFees = totalDisputesFees.add(treasuryBalance)
+  // Note that the total dispute fees could include already settled fees so we must discount them from the total treasury fees
+  const totalTreasuryFees = totalDisputesFees
+    .sub(totalSettledFees)
+    .add(treasuryBalance)
```

Comment:
Quick question on the treasury: it looks like its an array, so it must hold a number of tokens? Is this due to the fee token potentially changing, or is it because there could be multiple different tokens held by the treasury (e.g. ANJ and DAI)?

Reply:
Yes, it's the former. For example if a juror has some rewards in DAI that has not withdrawn form the treasury and the `feeToken` changes. We have a `TODO` there for handling potentially multiple tokens but probably we will need designs? Or we could just do a divider for every token.

Reply:
Gotcha, let's keep it as a todo (perhaps we can improve the TODO A bit :smile:)

Reply:
Will do 😄